uisdk/src/main/res/values-es-rES/strings.xml

```diff
@@ -27,24 +27,24 @@
     <string name="kh_uisdk_baggage_max">Máximo %d piezas de equipaje</string>
     <string name="kh_uisdk_base">Tarifa base</string>
     <string name="kh_uisdk_book_now">PAGAR Y RESERVAR</string>
-    <!--  Booking Confirmation  -->
+    <!-- Booking Confirmation -->
     <string name="kh_uisdk_booking_car">Confirmar viaje</string>
     <string name="kh_uisdk_booking_checkout_add_passenger">Agregar pasajero</string>
     <string name="kh_uisdk_booking_checkout_add_payment_method">Añadir método de pago</string>
     <string name="kh_uisdk_booking_checkout_add_payment_method_title">Método de pago</string>
-    <!--  Booking Checkout Component  -->
+    <!-- Booking Checkout Component -->
     <string name="kh_uisdk_booking_checkout_back_button">Volver</string>
     <string name="kh_uisdk_booking_checkout_edit_passenger">Editar</string>
     <string name="kh_uisdk_booking_checkout_passenger">Pasajero</string>
     <string name="kh_uisdk_booking_confirmation">Reserva confirmada</string>
     <string name="kh_uisdk_booking_details_error">Detalles de reserva inválidos</string>
     <string name="kh_uisdk_booking_error">Hubo un problema con tu solicitud de viaje. Inténtalo de nuevo</string>
-    <!--  Trip Allocation  -->
+    <!-- Trip Allocation -->
     <string name="kh_uisdk_booking_failed">Reserva fallida</string>
     <string name="kh_uisdk_booking_failed_body">La demanda es alta. %1 no tiene conductores disponibles en este momento</string>
     <string name="kh_uisdk_booking_failed_body_no_fleet_name">La demanda es alta en estos momentos. La flota seleccionada no tiene conductores disponibles.</string>
-    <!--  Terms and Conditions  -->
-    <string name="kh_uisdk_booking_terms">Al hacer una reserva, usted acepta las %1$s | %2$s así como los %4$s de %3$s | %5$s de %3$s.</string>
+    <!-- Terms and Conditions -->
+    <string name="kh_uisdk_booking_terms">Al hacer una reserva, usted  acepta las %1$s | %2$s así como los %4$s de %3$s | %5$s de %3$s.</string>
```

Comment:
Extra space before "acepta"

Reply:
Downloaded from Phrase, this is current value for this key.